.github/kokoro/ice40.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+CALLED=$_
```

Comment:
@mithro I think I'm missing the point of this. Why do we set this and not use it? Is this from a recipe/example?